frontend/src/main.tsx

```diff
@@ -7,6 +7,7 @@ import {
 } from "@tanstack/react-router";
 import React from "react";
 import ReactDOM from "react-dom/client";
+import { authenticatedRoute } from "./AuthenticatedRoute";
```

Comment:
```suggestion
import authenticatedRoute from "./AuthenticatedRoute";
```
Based on the above suggestion to fix the import.